manifests/init.pp

```diff
@@ -16,29 +16,49 @@
   $resolver_config_file_owner  = 'root',
   $resolver_config_file_group  = 'root',
   $resolver_config_file_mode   = '0644',
-) {
+  ){
 
-  # Validates domain
-  if is_domain_name($domain) != true {
-    fail("Domain name, ${domain}, is invalid.")
+  if $::kernel == 'windows' {
+    if is_array($nameservers) {
+      $stringifynameservers = join($nameservers,',')
+      $testserver = $nameservers[0]
```

Comment:
This only pulls the first nameserver from the array instead of setting all of the nameservers. Is that intended?